packages/sdks/output/react-native/src/blocks/button.lite.tsx

```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
 import { View, StyleSheet, Image, Text } from 'react-native';
-import { useContext } from 'react';
```

Comment:
These went away when updating to the new mitosis. They were bad/unused imports to begin with